galpy/orbit/orbit_c_ext/integrateFullOrbit.c

```diff
@@ -507,7 +507,7 @@ void parse_leapFuncArgs_Full(int npot,
       potentialArgs->Rforce= &RotateAndTiltWrapperPotentialRforce;
       potentialArgs->zforce= &RotateAndTiltWrapperPotentialzforce;
       potentialArgs->phiforce= &RotateAndTiltWrapperPotentialphiforce;
-      potentialArgs->nargs= (int) 2;
+      potentialArgs->nargs= (int) 10;
```

Comment:
I think it's possible that the reason that the C implementation is still not working (orbit integration hangs, or takes a longer time than my patience limit...) is something to do with this... its the only thing I can think of.

The compilation seems to work fine now.

Reply:
This should be the number of parameters that you're passing that are specific to the wrapper, which from `integrateFullOrbit.py` looks to be 10